css/styles.css

```diff
@@ -0,0 +1,68 @@
+/* GLOBAL STYLES */
+p {
+  font-family: 'Raleway', sans-serif;
+  font-weight: 300;
+}
+
+/* NAVBAR STYLES */
+.navbar-default {
+  background-color: transparent;
+  border-color: transparent;
+}
+.navbar-default .navbar-nav>.active>a,
+.navbar-default .navbar-nav>.active>a:focus,
+.navbar-default .navbar-nav>.active>a:hover {
+   color: #FFF;
+   background-color: rgba(17, 17, 17, .5);
+ }
+.navbar-default .navbar-nav>li>a {
+  color: #FFF;
+  text-shadow: 1px 1px 2px #0a0a0a;
+}
+.carousel-inner .item {
+  height: 60vh;
+  background-size: cover;
+  background-position: center;
+}
+.carousel-caption {
+  padding-bottom: 150px;
+}
+.first-slide {
+  background-image: url('../img/wood-2.jpg');
+}
+.slide-logo {
+  max-height: 300px;
+}
+.second-slide {
+  background-image: url('../img/restaurant-2.jpg');
+}
+.third-slide {
+  background-image: url('../img/farm-2.jpg');
+}
+
```

Comment:
Caressa, Rocky had us change the links and buttons to orange (#F8512F). Here is how I changed mine:
Links on the navbar: 
.navbar-default .navbar-nav>li>a:hover{
  color:#F8512F;
  background-color: rgba(17, 17, 17, .5);
}
buttons on the carousel:
.btn-primary,
.btn-primary:hover{
  background-color: #F8512F;
  border-color: #F8512F;
}